crates/env/build.rs

```diff
@@ -0,0 +1,23 @@
+use const_gen::*;
+use std::{
+    env,
+    fs,
+    path::Path,
+};
+
+fn main() {
+    // Default size of the buffer: 16 kB.
+    let mut size: usize = 1 << 14;
+    // if environmental variable is present we update the size.
+    if let Ok(size_str) = std::env::var("STATIC_BUFFER_SIZE") {
+        if let Ok(new_size) = size_str.parse::<usize>() {
+            size = new_size;
+        }
```

Comment:
Maybe it is worth printing an error or throwing it, because the user specified the `STATIC_BUFFER_SIZE`, but the value was incorrect